Documentation/kubelet-wrapper.md

```diff
@@ -51,12 +55,14 @@ Pods running in your cluster can reference remote storage volumes located on an
 
 ```ini
 [Service]
-Environment="RKT_OPTS=--volume iscsiadm,kind=host,source=/usr/sbin/iscsiadm --mount volume=iscsiadm,target=/usr/sbin/iscsiadm"
+Environment="RKT_OPTS=--uuid-file-save=/var/run/kubelet-pod.uuid --volume iscsiadm,kind=host,source=/usr/sbin/iscsiadm --mount volume=iscsiadm,target=/usr/sbin/iscsiadm"
```

Comment:
A nit for these documentation entries would be to multi-line this, and because it is documenting a specific functionality (e.g. iscsi in this case) - those flags should come first.